python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -579,6 +579,10 @@ def aten〇new_empty〡shape(self: List[int], size: List[int], dtype: Optional[i
 def aten〇new_empty_strided〡shape(self: List[int], size: List[int], stride: List[int], dtype: Optional[int] = None, layout: Optional[int] = None, device: Optional[device] = None, pin_memory: Optional[bool] = None) -> List[int]:
     return size
 
+def aten〇glu〡shape(self: List[int], dim: int = -1) -> List[int]:
+    self[dim] = self[dim] // 2
```

Comment:
This needs to be upstreamed first. See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_abstract_interpretation_functions.md#step-by-step-guide

> New shape functions should be added upstream following the example of https://github.com/pytorch/pytorch/pull/76889, though it can be useful to iterate locally in abstract_interp_lib_gen.py first.